Second version of ESSD figures

stephpenn1 opened the pull request:
- Map now uses Stamen maps for ease of reproducibility and includes point sizes to represent data density

![image](https://user-images.githubusercontent.com/20341158/79693764-a8cde000-823a-11ea-9208-d68e02a6cc03.png)

- New plot showing number of obs by DOY

![image](https://user-images.githubusercontent.com/20341158/79693787-c26f2780-823a-11ea-9856-c22d01b0c148.png)

stephpenn1 commented:
Still to do: 
- [ ] fix legend titles on map 
- [ ] add MAP/MAT plot

stephpenn1 commented:
@bpbond do not merge yet

stephpenn1 commented:
Thoughts? 

![image](https://user-images.githubusercontent.com/20341158/79781857-29104600-830c-11ea-82cc-f290fa6c3d83.png)

bpbond commented:
Ooh beautiful! Only possible suggestion might be COSORE points in a more contrasting color? Not sure.

________________________________
From: Stephanie Pennington <notifications@github.com>
Sent: Monday, April 20, 2020 1:38:28 PM
To: bpbond/cosore <cosore@noreply.github.com>
Cc: Bond-Lamberty, Benjamin <BondLamberty@pnnl.gov>; Mention <mention@noreply.github.com>
Subject: Re: [bpbond/cosore] Second version of ESSD figures (#214)


Thoughts?

[image]<https://protect2.fireeye.com/v1/url?k=75fe59bf-294b6770-75fe73aa-0cc47adc5e60-dc7f14021463489e&q=1&e=860fc16e-48e3-4aad-bb62-55ce81cccb4e&u=https%3A%2F%2Fuser-images.githubusercontent.com%2F20341158%2F79781857-29104600-830c-11ea-82cc-f290fa6c3d83.png>

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://protect2.fireeye.com/v1/url?k=a5e8bf0c-f95d81c3-a5e89519-0cc47adc5e60-d2d14a1ca711a819&q=1&e=860fc16e-48e3-4aad-bb62-55ce81cccb4e&u=https%3A%2F%2Fgithub.com%2Fbpbond%2Fcosore%2Fpull%2F214%23issuecomment-616706402>, or unsubscribe<https://protect2.fireeye.com/v1/url?k=bb9c9ba2-e729a56d-bb9cb1b7-0cc47adc5e60-5281d821ad282833&q=1&e=860fc16e-48e3-4aad-bb62-55ce81cccb4e&u=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAAO5U5HINZSTW3QSBRMCMLLRNSCBJANCNFSM4ML3SDQQ>.

stephpenn1 commented:
Everything is up to date. I am having trouble sourcing the `mat-map.R` script in the Rmd, otherwise this is ready to go

```
> source("mat-map.R")
Error in .getDataPath(path) : path does not exist: essd/
```

bpbond commented on essd/mat-map.R:
```diff
@@ -0,0 +1,59 @@
+library(RColorBrewer)
+
+# Download worldclim data for precip and tmean, store in essd/ folder
+precip <- getData("worldclim", path = "essd/", var="prec", res = 10)
```
If the WorldClim data are in `essd/`, and this script is there too, should the `path` parameter just be "./"?

stephpenn1 replied:
This function puts the WorldClim data in `essd/`. Since the working directory is just `cosore/`, the data goes there by default.

stephpenn1 replied:
@bpbond Everything is up to date on my end. I addressed your comment abt the path, see above. Also, lines 14 and 15 download the WorldClim data, so you shouldn't need my local files right?

bpbond commented:
@stephpenn1 When I try to knit the RMarkdown, I get an error in line 403, `data_meta` not found. 😕

stephpenn1 commented:
When I knit I get an error about not having the google sheet authors... but line 353 is...

`csr_table(table = 'data') %>% left_join(db) -> data_meta`

bpbond commented:
>line 353 is

That doesn't appear in the diff, so may not have gotten pushed?


![Screen Shot 2020-04-24 at 11 35 16 AM](https://user-images.githubusercontent.com/1956468/80230255-b19a2980-861f-11ea-8b72-da7b3bbcf6e6.png)

bpbond commented:
@stephpenn1 I'd like to turn back to this later this week...could you push up any code you have by then, and fx me needed WorldClim file(s)? Thank you.

stephpenn1 commented:
@bpbond Everything is up to date on my end. I addressed your comment abt the path, see above. Also, lines 14 and 15 download the WorldClim data, so you shouldn't need my local files right?

bpbond commented:
👏 @stephpenn1
